include/mbedtls/ecjpake.h

```diff
@@ -258,6 +258,29 @@ int mbedtls_ecjpake_derive_secret( mbedtls_ecjpake_context *ctx,
                             int (*f_rng)(void *, unsigned char *, size_t),
                             void *p_rng );
 
+/**
+ * \brief           Write the shared key material to be passed to a Key
+ *                  Derivation Function as described in RFC8236.
+ *
+ * \param ctx       The ECJPAKE context to use. This must be initialized,
+ *                  set up and have performed both round one and two.
+ * \param buf       The buffer to write the derived secret to. This must
+ *                  be a writable buffer of length \p len Bytes.
+ * \param len       The length of \p buf in Bytes.
+ * \param olen      The address at which to store the total number of Bytes
```

Comment:
That's a long & frustrating story. At some point another team, that integrated Mbed TLS into their product, and had pretty strict documentation rules, contributed a bunch of documentation PRs that we had to integrate on a tight deadline because Business Reasons™. Most of the changes were good, like being more explicit about sizes being in bits or bytes, a few were outright wrong because they misunderstood the existing documentation before rewriting it (which shows the existing documentation needed improvement anyway), and then there was this IMO stupid argument that they wanted to capitalize Bytes, because the abbreviation is capital B (as in kB)... So, due to the context we didn't have time to win this argument and make them re-submit with `bytes`, and had to focus instead on just fixing things that were semantically wrong. So there we are, with a bunch of wrongly-capitalized Bytes.

If you feel like raising a PR with `s/Byte/byte/g` everywhere, I think that's something we could do now. In the meantime, I think both `Bytes` and `bytes` can be accepted (the former because it's present everywhere, and the later because it's actually correct).